web-ui/src/main/resources/catalog/components/common/ows/OWSService.js

```diff
@@ -387,6 +387,10 @@
               //Add Info for Requests:
               if (capObj.Request) {
                 layers[i].capRequest = capObj.Request;
+              } 
+              
+              if(!layers[i].version) {
+                layers[i].version = capObj.version;                  
```

Comment:
Save the WFS version published by the GetCapabilities and use it to load the layer.